README.rst

```diff
@@ -78,6 +78,21 @@ or ``conda``::
 The documentation includes more detailed `installation instructions <http://scikit-learn.org/stable/install.html>`_.
 
 
+Changelog
+---------
+
+See the `changelog <http://scikit-learn.org/dev/whats_new.html>`__
+for a history of notable changes to scikit-learn.
+
+.. The following two lines are duplicated in doc/whats_new.rst
```

Comment:
I don't think comments like this in a README are appropriate. I'd rather remove this and the following reference to libraries.io. It is sufficient to have in what's new, or perhaps in installation instructions